### PR TITLE
Render content regardless of focus

### PR DESCRIPTION
**Pull request by kevinhowbrook**
issue #16

### CODE REVIEW

**Comment by jhoover4** [comment]
Can this be merged?

File: wagtailmarkdown/static/wagtailmarkdown/js/simplemde.attach.js [hunk 1]
```diff
@@ -32,3 +32,17 @@ $(document).ready(function() {
...
+
+/*
+ * Used to initialize content when MarkdownFields are used in admin panels.
+ */
+$(document).on('shown.bs.tab', function(e) {
+    $('.CodeMirror').each(function(i, el){
+        setTimeout(
+            function() {
+                el.CodeMirror.refresh();
+                console.log(el);
```
Comment (zerolab) [inline wagtailmarkdown/static/wagtailmarkdown/js/simplemde.attach.js]: this should definitely not be in the release

Comment (jhoover4) [reply wagtailmarkdown/static/wagtailmarkdown/js/simplemde.attach.js]: Okay then should the issue be closed?

Comment (zerolab) [reply wagtailmarkdown/static/wagtailmarkdown/js/simplemde.attach.js]: no, that was only meant for the `console.log()` line

Comment (jhoover4) [reply wagtailmarkdown/static/wagtailmarkdown/js/simplemde.attach.js]: Ah I didn't see that my bad 😬

Comment (kevinhowbrook) [reply wagtailmarkdown/static/wagtailmarkdown/js/simplemde.attach.js]: Removed, thanks Dan

**Comment by zerolab** [comment]
LGTM

**Pull request merged**